pkg/providers/aws/provider_redis.go

```diff
@@ -308,6 +312,20 @@ func (p *RedisProvider) createElasticacheCluster(ctx context.Context, r *v1alpha
 			errMsg := fmt.Sprintf("failed to add tags to elasticache: %s", msg)
 			return nil, types.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 		}
+		//cacheClusterOutput, err := cacheSvc.DescribeCacheClusters(&elasticache.DescribeCacheClustersInput{
+		//	CacheClusterId: cache.CacheClusterId,
+		//})
+		//if err != nil {
+		//	errMsg := fmt.Sprintf("error describing elasticache cluster %s", err)
+		//	return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
+		//}
+
+		//for _, foundCacheCluster := range cacheClusterOutput.CacheClusters {
+		//	logrus.Info(foundCacheCluster)
+		//	if r.Status.Version != *foundCacheCluster.EngineVersion {
+		//		r.Status.Version = *foundCacheCluster.EngineVersion
+		//	}
+		//}
```

Comment:
These commented out lines can be removed.

Reply:
spotted on rebase :+1: and removed